collector/slm.go

```diff
@@ -0,0 +1,401 @@
+// Copyright 2021 The Prometheus Authors
```

Comment:
```suggestion
// Copyright 2022 The Prometheus Authors
```
This needs to be updated in all of the new files.